src/native-common/AccessibilityUtil.ts

```diff
@@ -10,12 +10,15 @@
 
 import _ = require('./lodashMini');
 import React = require('react');
+import RN = require('react-native');
 
 import { AccessibilityUtil as CommonAccessibilityUtil, AccessibilityPlatformUtil } from '../common/AccessibilityUtil';
 
 import Types = require('../common/Types');
 
-const liveRegionMap: { [key: string]: string } = {
+type AccessibilityLiveRegionValue = 'none' | 'polite' | 'assertive';
+
+const liveRegionMap: { [key: string]: AccessibilityLiveRegionValue } = {
     [Types.AccessibilityLiveRegion.None]: 'none',
```

Comment:
No, it's not needed here because we're using a TypeScript-style map in this case. We couldn't use this approach in the other location because TypeScript maps support keys that are either strings or numbers, not other types.